app/components/WeatherForm.jsx

```diff
@@ -2,31 +2,60 @@ var React = require('react');
 var GoogleAutocomplete = require('./GoogleAutocomplete.jsx');
 
 var WeatherForm = React.createClass({
-  onFormSubmit: function(e) {
-    e.preventDefault();
-    var location = this.refs.googleAutoComplete.getLocation(); //set variable to location value
-    if (location.length > 0){ //check for input
+	onFormSubmit: function(e) {
```

Comment:
there were spaces before so I changed them to tabs
<img width="394" alt="screen shot 2016-10-10 at 11 26 28 am" src="https://cloud.githubusercontent.com/assets/2175676/19245063/7991e430-8edc-11e6-9606-56e0710c1cc5.png">